var/spack/repos/builtin/packages/pfunit/package.py

```diff
@@ -40,7 +41,13 @@ class Pfunit(CMakePackage):
 
     depends_on('python@2.7:', type=('build', 'run'))  # python3 too!
     depends_on('mpi', when='+mpi')
+    depends_on('m4', when='@4.1.5:4.1.9')
 
+    conflicts("%gcc@7.2.0", when="@4.0.0:",
+              msg='pFUnit v4 cannot be built with gcc 7.2.0. Please choose a different compiler.')
+    # See https://github.com/Goddard-Fortran-Ecosystem/pFUnit/pull/179
+    conflicts("+shared", when="@4.0.0:4.1.9")
+    conflicts("%gcc@:8.3.9", when="@4.0.0:")
```

Comment:
This conflict is redundant with the one on line 46. I'm guessing you're trying to say "it conflicts with GCC < Version('9')" but the spack versioning notation is pretty weird (especially with open/closed ranges). I think `%gcc@:8.999` might be a more standard way to write the restriction.

Reply:
Good catch!